src/cgame/cg_consolecmds.cpp

```diff
@@ -584,7 +584,6 @@ void CG_InitConsoleCommands()
 		trap_AddCommand( commands[ i ].cmd );
 	}
 
-	// Defined in src/engine/qcommon/q_shared.h, see BUTTON_ATTACK etc.
```

Comment:
Mention the new location?

Reply:
I'm not even sure what this comment is really meant to explain.

Reply:
The nth item in the list is the name of the "plus" command for the nth button. So it explains where the buttons are defined.

Reply:
Right, I'll try to make a better comment on that then.